pointsProgForm.php

```diff
@@ -24,37 +24,25 @@
     }
     if ($_SERVER["REQUEST_METHOD"] == "POST") {
```

Comment:
great design choices again, using the Event entity as a template and only keeping relevant variable names.